src/components/Search/utils.ts

```diff
@@ -205,8 +291,150 @@ export async function addExistingParamsToUrl(
+      keywords: a[1].buckets.map((b) => ({
+        label: b.key_as_string ?? b.key,
+        location: metadata.location,
+        filter: {
+          category: metadata.label,
+          location: metadata.location,
+          term: { value: b.key }
+        },
+        count: b.doc_count
+      }))
+    }
+  })
+}
+
+export const formatTermsConditionsVerifiedResults = (
+  aggregationResults: AggregationResult[]
+) => {
+  const termsConditionsVerified = aggregationResults.filter(
+    (ar) =>
+      ar.category === 'Is Verified' || ar.category === 'Terms and Conditions'
+  )
+  return {
+    category: 'Verified and Terms & Conditions',
+    keywords: termsConditionsVerified.flatMap((tcv) => {
+      const filters = {
+        ...tcv.keywords[0].filter,
+        category: 'Verified and Terms & Conditions',
+        term: { value: 'true' }
+      }
+      if (
+        tcv.keywords.find((a) => a.label === 'true') &&
+        tcv.category === 'Is Verified'
+      ) {
+        tcv.keywords = [
+          { ...tcv.keywords[0], label: 'verified', filter: filters }
+        ]
+      } else if (
+        tcv.keywords.find((a) => a.label === 'true') &&
+        tcv.category === 'Terms and Conditions'
+      ) {
+        tcv.keywords = [
+          { ...tcv.keywords[0], label: 'terms & conditions', filter: filters }
+        ]
+      }
+      return tcv.keywords
+    })
+  }
+}
+
+export const formatPriceResults = (
+  aggregationResults: AggregationResult[]
+): AggregationResult => {
+  const price: AggregationResult = aggregationResults.find(
+    (ar) => ar.category === 'Price'
+  )
+
+  const free = price.keywords
+    .filter((k) => +k.label === 0)
+    .reduce(
+      (a, b) => {
+        return {
+          label: a.label,
+          filter: {
+            category: a.filter.category,
+            ...b.filter,
+            location: b.filter.location,
+            term: { value: 0 }
+          },
+          count: a.count + b.count
+        }
+      },
+      { label: 'free', filter: { category: 'Price' }, count: 0 }
+    ) as Keyword
+  const paid = price.keywords
+    .filter((k) => +k.label > 0)
+    .reduce(
+      (a, b) => {
+        return {
+          label: a.label,
+          filter: {
+            category: a.filter.category,
+            location: b.filter.location,
+            range: [{ operation: 'gt', value: 0 }]
+          },
+          count: a.count + b.count
+        }
+      },
+      { label: 'paid', filter: { category: 'Price' }, count: 0 }
+    ) as Keyword
+
+  price.keywords = [free, paid]
+  return price
+}
+
+export const formatLanguagesResults = (
+  aggregationResults: AggregationResult[]
+): AggregationResult => {
+  const languages = aggregationResults.find((ar) => ar.category === 'Languages')
+  // eslint-disable-next-line array-callback-return
+  languages.keywords = languages?.keywords.map((lang) => {
+    if (typeof lang.label === 'string' && lang.label.includes('Dockerfile')) {
```

Comment:
should label not be a string by default? imo labels are always strings

Reply:
Fixed!